### PR TITLE
[Samples][Tableandlinkupdate]

### PR DESCRIPTION
**Pull request by KirtiPereira**
Updated the following files:

- TOC.yml
- conversation-basics.md

Removed the Bots and Messaging extension samples from the TOC.

@Wajeed-msft - please update the following before we approve the PR:

- [ ] Verify if the inline code has been updated.
- [ ] Does this file require a code snippet?

### CODE REVIEW

File: msteams-platform/tutorials/code-samples.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Microsoft Teams code samples
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
 keywords: Microsoft Teams developer samples
 ---
-# Tutorials and code samples for the Microsoft Teams developer platform
+# Tutorials for the Microsoft Teams developer platform
 
 Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
```
Comment (surbhigupta12) [inline msteams-platform/tutorials/code-samples.md:8]: Line 8, please change to:

Here you will find a list of tutorials that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.

File: msteams-platform/TOC.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@
     href: build-your-first-app/build-bot.md
   - name: Create a messaging extension
     href: build-your-first-app/build-messaging-extension.md
-- name: Tutorials and samples
+- name: Tutorials and code samples
   items:
   - name: Overview
```
Comment (surbhigupta12) [inline msteams-platform/TOC.yml:25]: Line 25, Please change "overview" to  "Tutorials"

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor comments.

**Comment by KirtiPereira** [comment]
@Wajeed-msft - gentle reminder to approve the PR. Thank you.

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**